zerver/lib/github.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+from zerver.lib.cache import cache_with_key
+
+
+@cache_with_key(lambda repo: f"latest_version:{repo}", timeout=60*60)
+def get_latest_release_version_from_github(repo: str) -> str:
+    try:
+        response = requests.get(f"https://api.github.com/repos/zulip/{repo}/releases")
+        data = response.json()
+        tag_name = data[0]["tag_name"]
+        if tag_name[0] == "v" or tag_name[0] == "V":
+            return tag_name[1:]
+        return tag_name
```

Comment:
This should use [`/repos/:owner/:repo/releases/latest`](https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-latest-release), which automatically gives you the right version instead of a list to sift through, and excludes pre-releases.

Reply:
Done.